tutorials/MixedHMC.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>This result seems a bit underwhelming compared to NumPyro's MixedHMC result. Is that expected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/facebookresearch/beanmachine/pull/1491/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
I think there is some tuning I still am not replicating. The result should match NumPyro's